cmd/main.go

```diff
@@ -269,3 +272,29 @@ func updateHealthMetricsPeriodically(nsxClient *nsx.Client) {
 		}
 	}
 }
+
+func checkLicense(nsxClient *nsx.Client, interval int, licenseTimeoutNoDFW int) {
+	err := nsxClient.ValidateLicense(true)
+	if err != nil {
+		os.Exit(1)
+	}
+	// if there is no dfw license enabled, check license more frequently
+	if !util.IsLicensed(util.FeatureDFW) {
+		if interval > licenseTimeoutNoDFW {
```

Comment:
If user set it in ConfigMap manually, I think we should use it

Reply:
done